### PR TITLE
Adds panels with scores for world and friends

### PR DESCRIPTION
**Pull request by vlidholt**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -367,6 +367,64 @@ class CenterArea extends StatelessComponent {
...
+
+  Widget _itemBuilder(BuildContext context, int item, int index) {
+    return new Stack(
+      [
+        new Positioned(
+          left: 0.0,
+          top: 0.0,
+          child: new Container(
+            width: 320.0,
+            height: 59.0,
```
Comment (abarth) [inline lib/main.dart]: Positioned takes a width and a height that you can use instead if you prefer.

File: lib/main.dart [hunk 2]
```diff
@@ -367,6 +367,64 @@ class CenterArea extends StatelessComponent {
...
+        new Positioned(
+          left: 70.0,
+          bottom: 12.0,
+          child: new Text(
+            "${(index + 1) * 25428}",
+            style: new TextStyle(fontSize: 16.0)
+          )
+        )
+      ],
+      key: new Key("${key.toString()}_$index")
```
Comment (abarth) [inline lib/main.dart]: You can just use `new Key(index)` if you want to use the index as a key.

**Comment by abarth** [comment]
LGTM

File: lib/main.dart (hunk 1 above)
Comment (vlidholt) [inline lib/main.dart]: Thanks, will change this

File: lib/main.dart (hunk 2 above)
Comment (vlidholt) [inline lib/main.dart]: Thanks, will change, wasn't sure if the keys needed to be globally unique.

**Pull request merged**